src/data/opy/constants.js

```diff
@@ -110,6 +110,12 @@ const opyConstants = {
             "isConstant": true,
             return: "string"
         },
+        "LOTS_OF_SPACES": {
+            "description": "Alias for Math.FUCKTON_OF_SPACES.",
```

Comment:
The point was to avoid the profanity.
```suggestion
            "description": "170 Em Spaces (U+2003). Add this to the end of a HUD text to ensure it will always be left-aligned.",
```